back/app/services/url_sanitization_service.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class UrlSanitizationService
+  private
+  class UrlScrubber < Rails::Html::PermitScrubber
+    URL_WHITELIST = [
+      %r{https:\/\/(.+)?(wistia.com|wi.st).*},
+      %r{https:\/\/.*\.typeform\.com\/to\/.*},
+      %r{https:\/\/widget\.surveymonkey\.com\/collect\/website\/js\/.*\.js},
+      %r{https:\/\/docs.google.com\/forms\/d\/e\/.*\/viewform\?embedded=true},
+      %r{https:\/\/surveys.enalyzer.com\/?\?pid=.*},
+      %r{https:\/\/www\.survey-xact\.dk\/LinkCollector\?key=.*},
+      %r{https:\/\/.*\.qualtrics\.com\/jfe\/form\/.*},
+      %r{https:\/\/www\.smartsurvey\.co\.uk\/.*},
+      %r{https:\/\/.*\.(microsoft|office)\.com\/},
+      %r{https:\/\/.*www\.eventbrite\.com\/static\/widgets\/*},
+      %r{https:\/\/.*arcgis\.com},
+      %r{https:\/\/public\.tableau\.com.*},
+      %r{https:\/\/.*datastudio\.google\.com\/embed*},
+      %r{https:\/\/app\.powerbi\.com\/},
+      %r{https:\/\/.*static\.ctctcdn\.com\/js*},
+      %r{https:\/\/.*instagram\.com},
+      %r{https:\/\/.*platform\.twitter\.com},
+      %r{https:\/\/name\.konveio\.com},
+    ].freeze
+
+    private_constant :URL_WHITELIST
+
+    def url_whitelisted?(url)
+      URL_WHITELIST.any? { |regex| regex.match? url }
+    end
```

Comment:
An option here might be to call the video_whitelisted function here instead, and check both the video and other URLs all at once. Then in the layout.rb we could just call this one function?

Reply:
I wouldn't do that because then URL whitelisting and video whitelisting would be tied together and that may be problematic if/when we need them separately. I would stick to the "1 class = 1 responsibility" principle.